app/assets/javascripts/short-form/GISService.js.coffee

```diff
@@ -2,37 +2,29 @@
 ####################################### SERVICE ############################################
 ############################################################################################
 
-GeocodingService = ($http, ShortFormDataService) ->
+GISService = ($http, $q, ShortFormDataService) ->
```

Comment:
Doesn't look like we actually need `$q` here--I don't see it used in the diff.

Reply:
👍